pkg/kademlia/routing_test.go

```diff
@@ -161,6 +161,43 @@ func TestConnectionSuccess(t *testing.T) {
 	}
 }
 
+func TestUpdateSelf(t *testing.T) {
+	id := teststorj.NodeIDFromString("AA")
+	rt, cleanup := createRoutingTable(t, id)
+	defer cleanup()
+	address := &pb.NodeAddress{Address: "a"}
+	node := &pb.Node{Id: id, Address: address}
```

Comment:
Let's add field `Type: pb.NodeType_STORAGE` to pb.Node{}. I'm working on making sure all references to pb.Node have a Type

Reply:
done